src/components/user/navBar/navBar.jsx

```diff
@@ -1,17 +1,44 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import { Icon } from '@iconify/react';
 import SearchBar from '../searchBar/searchBar';
 import './navBar.css'
 
 
 const NavBar = () => {
-    return ( 
-        <div>
-        <nav className='nav_body'>
-            <h2>Barra de Navegacion</h2>
-        </nav>
-            <SearchBar/>
+    return (
+        <div className='box'>
+            <nav className='nav_body'>
+                <Link to="/" className='link_nav'>
+                   <h2>Techno Trade</h2>
+                </Link>
```

Comment:
Hay algun motivo por el cual repitas el <Link to="/"...> varias veces en vez de poner todo adentro de uno unico?